dev/run-tests.py

```diff
@@ -283,12 +283,16 @@ def get_hadoop_profiles(hadoop_version):
     """
 
     sbt_maven_hadoop_profiles = {
-        "hadoop2.7": ["-Phadoop-2.7", "-Phive-1.2"],
-        "hadoop3.2": ["-Phadoop-3.2", "-Phive-2.3"],
+        "hadoop2.7": ["-Phadoop-2.7"],
+        "hadoop3.2": ["-Phadoop-3.2"],
     }
 
     if hadoop_version in sbt_maven_hadoop_profiles:
-        return sbt_maven_hadoop_profiles[hadoop_version]
+        if ("ghprbPullTitle" in os.environ and
```

Comment:
Can we do it consistently with what we handle Hadoop versions? (e.g., https://github.com/apache/spark/commit/8ab13065f626a12d18f6ac14a4dbf3d258919825#diff-3c9f4fccf7d30ce2e8fa86db2ad1fdad)

Reply:
and .. seems like we should change https://github.com/apache/spark/commit/c98e5eb3396a6db92f2420e743afa9ddff319ca2#diff-180360612c6b8c4ed830919bbb4dd459R56 too if this PR also targets to change default Hive profile to 2.3.